source/rust_verify_test/tests/state_machines.rs

```diff
@@ -5999,7 +5999,7 @@ test_verify_one_file! {
                 }
             }
         }}
-    } => Err(e) => assert_error_msg(e, "the first param to a 'readonly'")
```

Comment:
what's the reason for this change?

these errors all occur during the macro phase, so they aren't technically "VIR errors". What does `assert_vir_error_msg` actually check, now? Should it be renamed?

Reply:
ok, great, that distinction makes sense 👍

Reply:
we can no longer rely on getting the errors through a memory object, as we spawn `rust_verify` as a process for each test, so I had to make the distinction based on whether diagnostics carried an error code

Reply:
Can we use both the error code and a message?  I find things like `assert_rust_error_code(err, "E0308")` harder to read and less specific than `assert_error_msg(err, "cannot borrow 'x' as mutable more than once at a time")`.

Reply:
I switched back to just the message. If we prefer to have both, I'm happy to re-add them after I merge.